config.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 the generic-device-plugin authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"github.com/squat/generic-device-plugin/deviceplugin"
+	"gopkg.in/yaml.v3"
+	"k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+)
+
+const defaultDomain = "squat.ai"
+
+// InitConfig defines config flags, config file, and envs
+func InitConfig() error {
```

Comment:
For consistency, let's keep all types that are not meant to be used outside of the package (even if they mechanically cannot because it's the main package) lowercase
```suggestion
func initConfig() error {
```